### PR TITLE
feat: Reading from remote source (JsonBin)

### PR DESCRIPTION
**Pull request by travissleavitt**
- Serve static index.html that fetches settings (later we can add CRUD)

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+JSONBIN_API_KEY=
+JSONBIN_ROOT=
+JSONBIN_BINID=
```
Comment (travissleavitt) [inline .env.example:3]: Mainly for local. Need to update values on Heroku for live.

File: index.js [hunk 2]
```diff
@@ -119,6 +125,30 @@ app.post("/", (req, res) => res.send(run(req, "general")))
...
+        "secret-key": jsonBinApiKey,
+      },
+    })
+    const data = await response.json()
+    res.json(data)
+  } catch (error) {
+    // TODO: fallback on a static version
+    res.json(error)
+  }
+})
```
Comment (travissleavitt) [inline index.js:146]: Proxy the api call so we can hide the JSONBin key

File: index.js [hunk 3]
```diff
@@ -119,6 +125,30 @@ app.post("/", (req, res) => res.send(run(req, "general")))
...
+
+app.get("/", (req, res) => {
+  res.sendFile(path.join(`${__dirname}/static/index.html`))
+})
+
```
Comment (travissleavitt) [inline index.js:151]: This will become the edit CMS

**Comment by dgollub** [comment]
Approved! lol :-D
Good job!

File: .gitignore [hunk 4]
```diff
@@ -66,3 +66,6 @@ node_modules/
 
 # That Mac folder file thing
 .DS_Store
+
+.cache
+static
```
Comment (ewendlick) [inline .gitignore:71]: Should a `.env.production` entry go in here?

File: src/index.html [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html class="no-js" lang="">
```
Comment (ewendlick) [inline src/index.html:2]: Is Modernizr or something that uses this class used in this project?

File: src/js/index.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    })
+    const data = await response.json()
+    return data
+  } catch (error) {
+    console.log(error)
+    return {}
+  }
+}
+
+;(async () => {
```
Comment (ewendlick) [inline src/js/index.js:16]: I see you hiding there.

You little semicolon... thinking you can sneak past the linter by being at the start of the line...

**Review by ewendlick — APPROVED** [review]

File: src/index.html (hunk 5 above)
Comment (travissleavitt) [reply src/index.html:2]: Ha, I no. I just copied HTML5 Boilerplate.

**Pull request merged**